subscription_oca/tests/test_subscription_oca.py

```diff
@@ -680,7 +680,18 @@ def test_x_subscription_oca_pricelist_related_2(self):
         self.pricelist_l3.currency_id = self.env.ref("base.THB")
         self.sub_line.sale_subscription_id.pricelist_id = self.pricelist_l3
         res = self.sub_line._get_display_price(self.product_1)
-        self.assertAlmostEqual(int(res), 613)
+        self.assertAlmostEqual(self.pricelist_l3.currency_id.rate, 38.4070, 4)
+        self.assertAlmostEqual(self.pricelist1.currency_id.rate, 1.5289, 4)
+        self.assertAlmostEqual(
+            int(res),
+            int(
+                20.5
```

Comment:
@sergiocorato Isn't there a mean to get a fixed value (as test should not vary normally)?